src/libs/functions/formatRedirectLink.spec.ts

```diff
@@ -10,13 +10,13 @@ describe('formatRedirectLink', () => {
     })
   } as CozyClient
 
-  it('should format onboarding redirection', () => {
+  it('should format redirect link', () => {
     expect(formatRedirectLink('contacts/#/hash', client)).toStrictEqual(
       'http://contacts.mycozy.test/#/hash'
     )
   })
 
-  it('should format onboarding redirection', () => {
+  it('should format redirect link', () => {
```

Comment:
nit: can you add a unit test with the "wrong format" that we currently receive from the stack? This would make a good documentation of the current bug